test.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash -ex
 export VCOMPUTE=("${NODE_NAME}-Rinjin1","${NODE_NAME}-Rinjin2","${NODE_NAME}-Quanta")
+export UCSPE=("${NODE_NAME}-UCSPE")
```

Comment:
We should delete this file. It is either deprecated or not used at all. Should we delete in this PR?